assets/src/edit-story/components/keyboardShortcutsMenu/constants.js

```diff
@@ -34,4 +34,6 @@ export const SPECIAL_KEYS = {
   OPTION: { symbol: '⌥', title: __('Option', 'web-stories') },
   ALT: 'Alt',
   DELETE: __('Delete', 'web-stories'),
+  UP: { symbol: '⬆', title: __('Up arrow', 'web-stories') },
+  DOWN: { symbol: '⬇', title: __('Down arrow', 'web-stories') },
 };
```

Comment:
When I first loaded up the shortcuts window, this is how the up/down arrows are rendering on my machine (Mac, latest Chrome and latest Firefox):
![image](https://user-images.githubusercontent.com/40646372/93498922-3a637b80-f8c7-11ea-878e-7c04a0806678.png)

It looks like Wordpress is auto replacing the up/down arrow with emoji-images when it loads this script: `wp-emoji.js?ver=5.5.1`.  When I use chrome to block the url to that script, preventing WP from loading it, then the short cuts appear as we would expect:

![image](https://user-images.githubusercontent.com/40646372/93499207-9201e700-f8c7-11ea-9d62-5ff31f4b77e0.png)

Not sure if this is an issue, but just wanted to call it out.

Reply:
Oh, that's weird. I'm not seeing that - is that some plugin or built-in behavior? @swissspidy, do you know?

And @o-fernandez, is this an issue?

Reply:
It'd be great to not have that happen, but if it comes between shipping the feature with that issue or delaying, I'd ship and fix later. @swissspidy any thoughts on what may be happening?

Reply:
@barklund @o-fernandez  That's some built-in behavior in WordPress, yes. WordPress uses the [Twemoji](https://twemoji.twitter.com/) library to override emoji with images under some circumstances.

@barklund You can add a `wp-exclude-emoji` class name to prevent this from happening in this case.

Reply:
Oh, looks like that workaround only works on non-element nodes. https://wordpress.org/support/article/emoji/#converting-to-images

Reply:
An alternative would be to use small SVGs with proper aria-labels.

Reply:
Another alternative is to use "regular" (non-emoji) unicode arrows, but it doesn't look as good IMHO:

With `UPWARDS/DOWNWARDS ARROW`:
![Screen Shot 2020-09-18 at 11 01 52](https://user-images.githubusercontent.com/637548/93613240-99e08a80-f99e-11ea-8869-8e81f1797862.png)

With `UPWARDS/DOWNWARDS TRIANGLE-HEADED ARROW`:
![Screen Shot 2020-09-18 at 11 05 42](https://user-images.githubusercontent.com/637548/93613484-eb891500-f99e-11ea-9917-80457822d191.png)

In summary, I see the following options:

1. Use a small SVG (will take a little time, but not much - will make it more complex though)
1. Use one of the Unicode arrow sets above 
1. F*ck it, ship it.

@o-fernandez, your call!

Reply:
I think 1 or 3.
I don't really like how 2 looks, particularly if we keep Shift as it is now.

Reply:
Shipped as is.

Reply:
Just no...